src/interfaces/token.ts

```diff
@@ -162,6 +169,20 @@ export class TokenInterface<C extends ChainId> extends ServiceInterface<C> {
 
       const combinedVaultsAndIronBankTokens = this.mergeAddressables(vaultsTokens, ironBankTokens);
 
+      if (this.chainId === 250) {
+        combinedVaultsAndIronBankTokens.push({
+          address: ZeroAddress,
+          name: "Fantom",
+          dataSource: "fantom",
+          decimals: "18",
+          priceUsdc: "0",
```

Comment:
@xgambitox unsure where we can get the price from, do you know?

Reply:
We have a PricesHelper contract in mainnet https://etherscan.io/address/0x5d63a8584d91ebc5033d022afd6c5a7c7fddc99b#readContract

@jstashh do we have this one deployed on fantom?

Reply:
Try `this.yearn.services.oracle.getPriceUsdc(tokenAddress);`

Reply:
> Try `this.yearn.services.oracle.getPriceUsdc(tokenAddress);`

This doesn't seem to work, I'm getting `0` -- maybe that's not supported for Fantom yet.

We do have the Wrapped Fantom price, is it sensible to use that instead? Like so https://github.com/yearn/yearn-sdk/pull/262/commits/41622b15708d24bd5855a8b8189862a34c336bf8

Reply:
Yeah we can go with that, as price should always be ~same